tests/observability/helper/helper.go

```diff
@@ -0,0 +1,167 @@
+package helper
+
+import (
+	"fmt"
+
+	"github.com/test-network-function/cnfcert-tests-verification/tests/globalhelper"
+	"github.com/test-network-function/cnfcert-tests-verification/tests/globalparameters"
+	params "github.com/test-network-function/cnfcert-tests-verification/tests/observability/parameters"
+	"github.com/test-network-function/cnfcert-tests-verification/tests/utils/daemonset"
+	"github.com/test-network-function/cnfcert-tests-verification/tests/utils/deployment"
+	"github.com/test-network-function/cnfcert-tests-verification/tests/utils/pod"
+	"github.com/test-network-function/cnfcert-tests-verification/tests/utils/statefulset"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+
+	. "github.com/onsi/ginkgo"
```

Comment:
Ideally we want to use ginkgo api only in tests files and test suite files. Can you refactor code

Reply:
You need to return errors from all funcs and process them on the test level

Reply:
Done. I've removed the helper functions.